prometheus_client/__init__.py

```diff
@@ -17,7 +17,6 @@
 
 CONTENT_TYPE_LATEST = exposition.CONTENT_TYPE_LATEST
 generate_latest = exposition.generate_latest
-MetricsHandler = exposition.MetricsHandler
```

Comment:
Removing this is a breaking change, please keep it around.

Reply:
Ok. I think we need a test in that case.

Reply:
I don't feel a strong need for a test for this.

Reply:
Explicit is better than implicit ;) my 2¢